samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -66,6 +66,7 @@ const locales = {
 	'th-TH - Thai, with tallglyph characters': 'th-TH',
 	'ar-SA - Arabic, RTL and standard font': 'ar-SA',
 	'ur-PK - Urdu, RTL and custom Urdu font': 'ur-PK',
+	'he-IL - Hebrew, RTL and standard font': 'he-IL',
```

Comment:
Let's move `he-IL` to `am-ET` below and like
'he-IL - Hebrew, RTL, no meridiem': 'he-IL', looks better.